setup.cfg

```diff
@@ -13,7 +13,7 @@ filterwarnings =
     ignore::sqlalchemy.exc.SAWarning
 addopts =
     --cov
-mock_use_standalone_module = true
+mock_use_standalone_module = false
```

Comment:
You can remove the line completely.

Reply:
Will do, also to trigger Travis (it is still queued).